internal/operators/ocs/ocs_operator.go

```diff
@@ -106,7 +106,7 @@ func (o *operator) ValidateHost(ctx context.Context, cluster *common.Cluster, ho
 }
 
 // GenerateManifests generates manifests for the operator
-func (o *operator) GenerateManifests(cluster *common.Cluster) (map[string][]byte, error) {
+func (o *operator) GenerateManifests(_ *common.Cluster) (map[string][]byte, error) {
```

Comment:
Does this need to be changed in this PR?

Reply:
we aren't using this variable anywhere so I changed it, should I open a different PR for this?

Reply:
yes, it is best to only have the changes relevant to the PR.

Reply:
Okay, I will update it.